src/pages/RiskTracker.tsx

```diff
@@ -0,0 +1,183 @@
+import React from 'react'
+import { Alert, Button } from 'react-bootstrap'
+import { useTranslation } from 'react-i18next'
+import { BsBoxArrowUpRight } from 'react-icons/bs'
+import { Link, useParams } from 'react-router-dom'
+
+import activityModeling from './img/activity-modeling.png'
+import podOverviewIntro from './img/budget-overview.png'
+import { DropdownNav } from 'components/DropdownNav'
+import { MarkdownContents } from 'components/markdown/PaperPage'
+import { mailchimpLink, spreadsheetUrl } from 'components/RiskTrackerUtil'
+import { TableOfContents } from 'components/TableOfContents'
+import { pages } from 'posts/tracker/index'
+
+export const RiskTrackerDocumentation = (): React.ReactElement => {
+  const { id } = useParams()
+  const { t } = useTranslation()
+  return (
+    <MarkdownContents
+      posts={pages}
+      id={id}
+      tableOfContentsPageTitle={t('menu.risk_tracker.start_here')}
+      sectionTitle={t('menu.risk_tracker.header')}
+    />
+  )
+}
+
+export const SubscribeForm: React.FunctionComponent = () => {
+  return (
+    <>
+      <h2>Subscribe for updates</h2>
+      <p>
+        We will send email updates when we make feature upgrades and bug fixes
+        to the Risk Tracker.
+      </p>
+      <p>{mailchimpLink()}</p>
+    </>
+  )
+}
+
+export const RiskTrackerTOC = (): React.ReactElement => {
+  return (
+    <TableOfContents posts={pages} title="Risk Tracker" baseNavPath="/tracker">
+      <hr />
+      <div>
+        <h2>What is the Risk Tracker for?</h2>
+        <p>
+          The microCOVID Risk Tracker is a spreadsheet that lets you estimate
+          the risk of getting COVID from an activity or relationship in your
+          daily life, using the{' '}
+          <Link to="/paper/research-source">best research available</Link>.
+        </p>
+        <img
+          src={podOverviewIntro}
+          alt="Pod overview"
+          style={{ width: '800px', maxWidth: '100%' }}
+        />
+        <p>
+          The Risk Tracker can help you model the risk of activities and people
+          you are seeing more precisely than is possible with the{' '}
+          <Link to="/calculator">website calculator</Link>. You can track log
+          your activities over time to keep your activities within a risk budget
+          that you define to be safe.
+        </p>
+        <p>
+          <strong>The Risk Tracker might be a good fit for you if:</strong>
```

Comment:
The Risk Tracker might be a good fit if you want to:

Reply:
✅